tle/util/cache_system2.py

```diff
@@ -331,11 +339,24 @@ async def _fetch_problemsets(self, contests, *, force_fetch=False):
 
     async def _fetch_for_contest(self, contest_id):
         try:
-            _, problemset, _ = await cf.contest.standings(contest_id=contest_id, from_=1,
+            contest, problemset, _ = await cf.contest.standings(contest_id=contest_id, from_=1,
                                                           count=1)
+            contest_name: str = contest.name
+            contest_name = contest_name.lower()
+            divisions = []
+            for i in range (1, 5):
+                if f'div. {i}' in contest_name:
```

Comment:
maybe we can increase the robustness if we steal some code from ;vc command
`        recommendations = {contest.id for contest in contests if
                           contest.matches(markers)`
where markers is like our tags. so we can just iterate over the 4 divisions and check 
`if contest_name.matches("div1") `
and ig it works

Reply:
matches is a member function on the Contest object